modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -2104,9 +2104,6 @@ export class Wallet implements IWallet {
    * Create a staking wallet from this wallet
    */
   toStakingWallet(): StakingWallet {
-    if (!this.baseCoin.supportsStaking()) {
```

Comment:
this method is no longer used anywhere, please also remove from baseCoin & all the other coins that override it.

Reply:
Fixed.